CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* **Overall**
+  * [CHANGE] v4m-metrics and v4m-logs Helm chart will now encrypt passwords that are stored in Helm values.
```

Comment:
Helm terminology is very confusing.  Technically, the name of the chart is "v4m" and "v4m-metrics" and "v4m-logs" are the names of _releases_ of that chart.  And, we aren't actually _encrypting_ the passwords we are just replacing them with asterisks.  So, maybe something like:
[CHANGE] Actual passwords are replaced with asterisks in deployed instances of the v4m Helm chart (e.g. v4m-logs, v4m-metrics, etc.).

Reply:
Yes, it is.  I updated with your suggestion in this next push.